contracts/bridges/BaseBridgeReceiver.sol

```diff
@@ -79,4 +79,14 @@ contract BaseBridgeReceiver {
 
         emit ProcessMessage(messageSender, targets, values, signatures, calldatas, eta);
     }
+
+    function executeTransaction(
```

Comment:
open question: should this function be callable by anyone?

if not, then who should be allowed to call it?